### PR TITLE
Introduce Multi-token support on Portfolio page

### PR DESCRIPTION
**Pull request by maxschnaider**

### CODE REVIEW

File: src/app/portfolio/components/PortfolioHistoryTradeItem.tsx [hunk 1]
```diff
@@ -52,7 +59,8 @@ export const PortfolioHistoryTradeItem = ({
       <Td px={2}>
         <Text color={trade.outcomeIndex == 0 ? 'green' : 'red'} fontWeight={'bold'}>
           {market?.outcomeTokens[trade.outcomeIndex ?? 0]}{' '}
-          {NumberUtil.toFixed(trade.outcomeTokenPrice, 3)} {collateralToken.symbol}
+          {NumberUtil.toFixed(trade.outcomeTokenPrice, 3)}{' '}
```
Comment (oleksandrtarapata) [inline src/app/portfolio/components/PortfolioHistoryTradeItem.tsx:62]: this method displays incorrect denomination. Consider to use **NumberUtil.formatThousands**

File: src/app/portfolio/components/PortfolioPositionCard.tsx [hunk 2]
```diff
@@ -44,98 +59,139 @@ export const PortfolioPositionCard = ({ position, ...props }: IPortfolioPosition
...
+            <HStack>
+              <Text>
+                {`${NumberUtil.toFixed(position.collateralAmount, 4)} ${
+                  position.market.collateral?.symbol
+                }`}
+              </Text>
+
+              <Text fontSize={'12px'} color={'fontLight'}>
+                ~$
+                {NumberUtil.toFixed(
```
Comment (oleksandrtarapata) [inline src/app/portfolio/components/PortfolioPositionCard.tsx:127]: same as here

https://github.com/limitless-labs-group/Limitless-Exchange/pull/63/files#r1601224364

File: src/app/portfolio/components/PortfolioPositionCard.tsx [hunk 3]
```diff
@@ -44,98 +59,139 @@ export const PortfolioPositionCard = ({ position, ...props }: IPortfolioPosition
...
+              )} ${position.market.collateral?.symbol}`}
+            </Text>
+          </HStack>
+
+          {/* Max win Row */}
+          <HStack w={'full'} justifyContent={'space-between'}>
+            <Text color={'fontLight'}>Max win</Text>
+
+            <HStack>
+              <Text>{`${NumberUtil.toFixed(position.outcomeTokenAmount, 4)} ${
```
Comment (oleksandrtarapata) [inline src/app/portfolio/components/PortfolioPositionCard.tsx:155]: same https://github.com/limitless-labs-group/Limitless-Exchange/pull/63/files#r1601224364

File: src/app/portfolio/components/PortfolioPositionCard.tsx [hunk 4]
```diff
@@ -44,98 +59,139 @@ export const PortfolioPositionCard = ({ position, ...props }: IPortfolioPosition
...
+                position.market.collateral?.symbol
+              }`}</Text>
+
+              <Text fontSize={'12px'} color={'fontLight'}>
+                ~$
+                {NumberUtil.toFixed(
```
Comment (oleksandrtarapata) [inline src/app/portfolio/components/PortfolioPositionCard.tsx:161]: this one too 

https://github.com/limitless-labs-group/Limitless-Exchange/pull/63/files#r1601224364

**Comment by valentynkvasov** [comment]
@maxschnaider pls merge it to dev when all comments fixed

**Pull request merged**